src/opentype.mjs

```diff
@@ -533,51 +444,20 @@ function parseBuffer(buffer, opt={}) {
  * We use the node.js callback convention so that
  * opentype.js can integrate with frameworks like async.js.
  * @alias opentype.load
- * @param  {string} url - The URL of the font to load.
- * @param  {Function} callback - The callback.
+ * @deprecated
  */
-function load(url, callback, opt = {}) {
-    const isNode = typeof window === 'undefined';
-    const loadFn = isNode && !opt.isUrl ? loadFromFile : loadFromUrl;
-
-    return new Promise((resolve, reject) => {
-        loadFn(url, function(err, buffer) {
-            if (err) {
-                if (callback) {
-                    return callback(err);
-                } else {
-                    reject(err);
-                }
-            }
-            let font;
-            try {
-                font = parseBuffer(buffer, opt);
-            } catch (e) {
-                if (callback) {
-                    return callback(e, null);
-                } else {
-                    reject(e);
-                }
-            }
-            if (callback) {
-                return callback(null, font);
-            } else {
-                resolve(font);
-            }
-        });
-    });
+function load() {
+    console.error('DEPRECATED! migrate to: opentype.parse(buffer, opt) See: https://github.com/opentypejs/opentype.js/issues/675');
 }
 
 /**
  * Synchronously load the font from a URL or file.
  * When done, returns the font object or throws an error.
  * @alias opentype.loadSync
- * @param  {string} url - The URL of the font to load.
- * @param  {Object} opt - opt.lowMemory
- * @return {opentype.Font}
+ * @deprecated
  */
-function loadSync(url, opt) {
```

Comment:
I thought we agreed that this would still function in 2.0.0 with a warning, and be removed in a subsequent release.